homeassistant/components/ddwrt/device_tracker.py

```diff
@@ -103,16 +107,20 @@ def _update_info(self):
         """
         _LOGGER.info("Checking ARP")
 
-        url = '{}://{}/Status_Wireless.live.asp'.format(
-            self.protocol, self.host)
+        endpoint = 'Wireless' if self.wireless_only else 'Lan'
```

Comment:
So hitting the LAN end-point returns the ARP table that includes all the wireless devices too?

Reply:
Yes, that's correct. All wireless devices are included in LAN but not all LAN devices are included in wireless.